src/sentry/sentry-error-interceptor.ts

```diff
@@ -21,16 +21,26 @@ export class SentryErrorInterceptor implements NestInterceptor {
           if (!withScope || !captureException) return;
 
           withScope((scope) => {
-            const data = Handlers.parseRequest(
-              {} as any,
-              context.switchToHttp().getRequest(),
-              {},
-            );
+            const contextType = context.getType();
 
-            scope.setExtra('req', data.request);
+            if (contextType === 'http') {
+              const data = Handlers.parseRequest(
+                {} as any,
+                context.switchToHttp().getRequest(),
+                {},
+              );
+
+              scope.setExtra('http_req', data.request);
+
+              if (data.extra) scope.setExtras(data.extra);
+              if (data.user) scope.setUser(data.user);
+            }
+
+            if (contextType === 'rpc') {
+              scope.setExtra('context_rpc', context.switchToRpc().getContext());
+              scope.setExtra('data_rpc', context.switchToRpc().getData());
+            }
```

Comment:
@Harasz It looks like this is new code? Just curious, what is the motivation for it?

Reply:
This will give use more information about failed RPC request. It is already implemented in KYC app, I can show you how its looks like in Sentry dashboard.

Reply:
Ah okay, thanks. Are we making `rpc` requests?

Reply:
Yes, in nestjs RPC request is that coming from micro-services, in our case from NATS.

Reply:
Ah, news to me, thanks for the info 🙏